e2e/app.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { test, expect } from '@playwright/test'
+import { setupClerkTestingToken } from '@clerk/testing/playwright'
+
+const URL_SIGN_IN = '/sign-in'
+const URL_ADMIN = '/admin'
+const URL_PROFILE = '/admin/profile'
+const URL_PROTECTED_ROUTE = '/some/random/protected-route/foo'
+const URL_ROOT = '/'
+const SERVER_SECRET = 'SvelteKit is awesome'
+
+test('User starts logged out', async ({ page }) => {
+	await page.goto('/')
+	const signIn = page.getByTestId('sign-in')
+	await expect(signIn).toBeVisible({ timeout: 30_000 })
+})
+
+test('User cannot access admin if not logged in', async ({ page }) => {
+	await page.goto(URL_ADMIN)
+	await page.waitForURL(URL_SIGN_IN)
+	await page.goto(URL_PROFILE)
+	await page.waitForURL(URL_SIGN_IN)
+})
+
+test('User cannot access function-based protected route if not logged in', async ({ page }) => {
+	await page.goto(URL_PROTECTED_ROUTE)
+	await page.waitForURL(URL_SIGN_IN)
+})
+
+test.describe('app', () => {
+	test('sign in', async ({ page }) => {
+		await setupClerkTestingToken({ page })
+
+		// Navigate to sign in page.
+		await page.goto(URL_ROOT)
+		const signInButton = page.getByTestId('sign-in')
+		await signInButton.click()
+
+		// Sign in with email and password
+		await page.waitForURL(URL_SIGN_IN)
+		await expect(page.locator('h1')).toContainText('Sign in')
+		await page.waitForSelector('.cl-signIn-root', { state: 'attached' })
+		await page.locator('input[name=identifier]').fill(process.env.E2E_CLERK_USER_USERNAME!)
+		await page.getByRole('button', { name: 'Continue', exact: true }).click()
+		await page.locator('input[name=password]').fill(process.env.E2E_CLERK_USER_PASSWORD!)
+		await page.getByRole('button', { name: 'Continue', exact: true }).click()
+
+		// Arrive at the admin page.
+		await page.waitForURL('/admin')
+		const serverSecret = page.getByTestId('server-secret')
+		await expect(serverSecret).toContainText(SERVER_SECRET)
+
+		// Can see the user button.
+		const userButton = page.getByTestId('user-button')
```

Comment:
Is this still being expected? Just downloaded the code and tried to run the tests:

Error: Timed out 5000ms waiting for expect(locator).toBeVisible()

    Locator: getByTestId('user-button')
    Expected: visible
    Received: <element(s) not found>

Reply:
I think it can be related to the organization switcher throwing an error because organizations are not enabled on my account...

Reply:
> Is this still being expected? Just downloaded the code and tried to run the tests:
> 
> Error: Timed out 5000ms waiting for expect(locator).toBeVisible()
> 
> ```
> Locator: getByTestId('user-button')
> Expected: visible
> Received: <element(s) not found>
> ```

Yes

<img width="462" alt="Screenshot 2024-09-27 at 8 13 22 AM" src="https://github.com/user-attachments/assets/27065ab9-d052-4125-8894-7325f8dacaad">